webapp/channels/src/components/admin_console/data_grid/data_grid_row.tsx

```diff
@@ -37,18 +42,18 @@ const renderCell = (row: Row, column: Column) => {
             {row.cells[column.field]}
         </div>
     );
-}
+};
 
-const DataGridRow = ({ row, columns}: Props) => {
-    const cells = columns.map((col) => renderCell(row, col));
+const DataGridRow = ({row, columns}: DataGridRowProps) => {
+    const cells = columns.map((column) => dataGridCell({row, column}));
```

Comment:
Since it is a component, use component notation:
```suggestion
    const cells = columns.map((column) => (
        <DataGridCell
            row={row}
            column={column}
        />
    ));
```

Reply:
Applied with an addition of `key` prop to `DataGridCell`